yamllint/cli.py

```diff
@@ -50,6 +50,11 @@ def supports_color():
             hasattr(sys.stdout, 'isatty') and sys.stdout.isatty())
 
 
+def is_github_actions():
+    if 'GITHUB_ACTIONS' in os.environ and 'GITHUB_WORKFLOW' in os.environ:
+        return True
```

Comment:
```suggestion
    return 'GITHUB_ACTIONS' in os.environ and 'GITHUB_WORKFLOW' in os.environ
```

Reply:
dropped this function in favor of your suggestion to move it to the format selection block